assets/scripts/google-analytics-opt-out.js

```diff
@@ -12,9 +12,15 @@ for (var i = 0; i < window.gaIdList.length; i++) {
  */
 function gaOptout() {
   var disableStr;
+  var domainName;
+  if (location.host.indexOf('www.') === 0) {
```

Comment:
As mentioned previously, the code should use the cookie domain of the backend, because this can be configured explicitly to be e.g. the second level domain (e.g., example.com) when the site runs on a third or fourth level domain (e.g., shop.example.com or my.shop.example.com), in order to make the cookies (primarily the user session) available to other applications.

See https://codex.wordpress.org/Editing_wp-config.php#Set_Cookie_Domain

If you do not know how to pass a value from the backend to the frontend as a JavaScript setting, then we can move forward here by using the value of `document.domain` as an interim solution, even though that value is not really correct in all cases.

Implementing the final fix (the correct cookie domain from the backend) would be preferred.